tests/unit/BloomFilterSyncTest.cpp

```diff
@@ -0,0 +1,127 @@
+/* This code is part of the GenSync project developed at Boston University.  Please see the README for use and references. */
+//
+// Created by Anish Sinha on 12/4/23.
+//
+
+#include "BloomFilterSyncTest.h"
+#include <GenSync/Syncs/GenSync.h>
+#include <GenSync/Syncs/BloomFilterSync.h>
+#include "../TestAuxiliary.h"
+CPPUNIT_TEST_SUITE_REGISTRATION(BloomFilterSyncTest);
+
+BloomFilterSyncTest::BloomFilterSyncTest() = default;
+
+BloomFilterSyncTest::~BloomFilterSyncTest() = default;
+
+void BloomFilterSyncTest::setUp() {
+	const int SEED = 93;
+	srand(SEED);
+}
+
+void BloomFilterSyncTest::tearDown() {
+}
+
+void BloomFilterSyncTest::BloomFilterSyncSetReconcileTest() {
+	const int BITS = sizeof(randZZ());
```

Comment:
Why sizeof?

Reply:
The BITS constant tells the Sync Protocol the size of elements being inserted. Since we are inserting randZZ(), we must set the element size to its size. This is implementation is used in the other tests as well such as the Cuckoo Sync Test.

Reply:
Please check to make sure that sizeof(randZZ()) is reporting something related to the actual size of the integers.  sizeof typically only tells you about the memory usage.